src/collections/Form/FormField.js

```diff
@@ -93,7 +93,7 @@ function FormField(props) {
 
   return (
     <ElementType className={classes}>
-      {createHTMLLabel(label)}
+      {createHTMLLabel(label, { htmlFor: controlProps && controlProps.id })}
```

Comment:
We'll need a test that asserts the `htmlFor` prop is added when an `id` is passed to `FormField`.  You can find tests in `test/specs/collections/Form/FormField-test.js`.

Let's also add this test to each subcomponent usage (`Form.Input`, etc).  These tests would go in `test/specs/collections/Form/Form-test.js`.

Because this test will be repeated several times, the test itself should be added only once to `commonTests.js`.  You'll see several examples of how to write a common test there.  Just add it to the bottom.  You'll see several examples of how to use a common test in the Form tests I've noted above.